test/integration/model-name.integration.js

```diff
@@ -0,0 +1,35 @@
+// Copyright IBM Corp. 2016,2018. All Rights Reserved.
+// Node module: loopback-connector-kv-redis
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+'use strict';
+
+const createDataSource = require('../helpers/data-source-factory');
+const expect = require('../helpers/expect');
+
+describe('modelName', function() {
+  it('returns default model name', () => {
+    const ds = createDataSource();
+    const modelName = 'TestModel';
+    const Model = ds.createModel(modelName);
+
+    const resolvedModel = ds.connector.modelName(modelName);
```

Comment:
```suggestion
    const resolvedModelName = ds.connector.modelName(modelName);
```

The same comment applies to the test below too.